frontend/src/hooks/usePythonSyntax.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-useless-escape */
 import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
 
-export const pythonSyntaxDefinition: monaco.languages.IMonarchLanguage = {
+const pythonSyntaxDefinition: monaco.languages.IMonarchLanguage = {
```

Comment:
@denishacquin do we really need a hook for something that doesn't dynamically change?

Reply:
good point, moving it to a constant